src/components/case_manager/PersonTable.tsx

```diff
@@ -348,39 +314,25 @@ export default function EnhancedTable<
               />
             )}
             {visibleRows.map((row) => {
-              const isItemSelected = isSelected(row.email);
               const labelId = row.email;
 
               return (
                 <StyledTableRow
                   hover
-                  role="checkbox"
-                  aria-checked={isItemSelected}
+                  role="link"
                   tabIndex={-1}
                   key={row.email}
-                  selected={isItemSelected}
                   sx={{ cursor: "pointer" }}
+                  onClick={() =>
+                    handleLinkToPage(
```

Comment:
can this not be a `<Link/>`?

Reply:
The <a> tag can't be a child of the table element, preventing me from wrapping the row in a Link. There are some other workarounds like changing the root component of the table to divs. 

Keep it as is: Unable to right click/open in new tab. Keep the semantics of tables.
Change it to divs: Possible issue with semantics?

Reply:
https://stackoverflow.com/a/71370700

Reply:
ah, I see, my bad